### PR TITLE
Add React Bootstrap to improve look and updated result list logic

### PR DESCRIPTION
**Pull request by troelze**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,3 @@
-REACT_APP_REDIRECT_URL=http://localhost:3000/redirect
+REACT_APP_REDIRECT_URL=http://localhost:3001/redirect
```
Comment (troelze) [inline .env:1]: y tho?

Comment (elimchayseng) [reply .env:1]: my b  - will revert

File: src/pages/YourDistance.js [hunk 2]
```diff
@@ -1,25 +1,54 @@
...
-    console.log(user.data)
-    return (
-        <div>
-            <h1>Hi, {returnTokens.athlete.firstname}!</h1>
-            <h2>Your total run distance: {user.data.all_run_totals.distance}m</h2>
-            <h2>Your total ride distance: {user.data.all_ride_totals.distance}m</h2>
-            <h2>Your total swim distance :{user.data.all_swim_totals.distance}m</h2>
-        </div>
-    );
+  //console.log(Object.entries(user.data));
```
Comment (troelze) [inline src/pages/YourDistance.js:6]: delete

Comment (troelze) [reply src/pages/YourDistance.js:6]: nvm

**Comment by glimchayseng** [comment]
Can you post a picture of what this looks like?

**Pull request closed without merging**